www/front_src/src/components/navigation/index-old.js

```diff
@@ -0,0 +1,303 @@
+import React, { Component } from "react";
```

Comment:
Is this file still needed? otherwise I think we can get rid of it

Reply:
I have absolutely no idea :) @danielhalupka ?

Reply:
I removed it